python/proton/_message.py

```diff
@@ -430,6 +430,31 @@ def _set_reply_to_group_id(self, value):
         :raise: :exc:`MessageException` if there is any Proton error when using the setter.        
         """)
 
+    def _get_instructions(self):
+        return self.instruction_dict
+    def _set_instructions(self, instructions):
+        if isinstance(instructions, dict):
+            self.instruction_dict = AnnotationDict(instructions, throw=False)
```

Comment:
"Throw" seems like a Java-ism.

Reply:
Changed to "raise_on_error", as "raise" on its own in reserved in Python.